addons/mail_bot/models/mail_bot.py

```diff
@@ -288,16 +292,6 @@ def _body_contains_emoji(self, body):
             return True
         return False
 
-    def _is_bot_pinged(self, values):
-        odoobot_id = self.env['ir.model.data']._xmlid_to_res_id("base.partner_root")
-        return odoobot_id in values.get('partner_ids', [])
-
-    def _is_bot_in_private_channel(self, record):
-        odoobot_id = self.env['ir.model.data']._xmlid_to_res_id("base.partner_root")
-        if record._name == 'discuss.channel' and record.channel_type == 'chat':
-            return odoobot_id in record.with_context(active_test=False).channel_partner_ids.ids
-        return False
```

Comment:
Note: cleanup, used only once or written in much more complex ways than necessary.